doc/uploads/process.adoc

```diff
@@ -0,0 +1,38 @@
+[#head]
+# Process from the upload to the fetch
+
+This documentation explains what happens after uploading the zip containing the details about the build.
+It is highly recommended to take a look to the <<HowItWorks.adoc#head, How It Works>> section before.
+
+## Uploading the zip
+
+### ExecutionResource.java
+-> First step is to upload the zip (https://your-ara-server.org/api/projects/cube-back/executions/upload).
+
+### ExecutionService.java
+-> The zip is then unzipped +
+-> All the (cycle and build) details are extracted +
+-> These details are saved in the following path: /path/in/your/server/__{{project}}__/__{{branch}}__/__{{cycle}}__ +
+-> The builds are then indexed
+
+### ExecutionCrawler.java
+
+-> Choose the fetcher (see below)
+
+### FetcherService.java
+
+This service enables to get the builds details. It can declined into several strategies:
+
+* HttpFetcher.java: getting the builds from a remote server
+* FileSystemFetcher.java: getting the builds locally
+* OldFileSystemFetcher.java: also getting the builds locally
```

Comment:
You can remove this one, it's deprecated and should be removed in 5.0.0

Reply:
done